import PureCloudPlatformClientV2
from PureCloudPlatformClientV2.rest import ApiException
from pprint import pprint

# Configure OAuth2 access token for authorization: PureCloud OAuth
PureCloudPlatformClientV2.configuration.access_token = 'your_access_token'
# or use get_client_credentials_token(...), get_saml2bearer_token(...) or get_code_authorization_token(...)

# create an instance of the API class
api_instance = PureCloudPlatformClientV2.OutboundApi();
callable_time_set_id = 'callable_time_set_id_example' # str | Callable Time Set ID
body = PureCloudPlatformClientV2.CallableTimeSet() # CallableTimeSet | DialerCallableTimeSet

try:
    # Update callable time set
    api_response = api_instance.put_outbound_callabletimeset(callable_time_set_id, body)
    pprint(api_response)
except ApiException as e:
    print("Exception when calling OutboundApi->put_outbound_callabletimeset: %s\n" % e)